src/android/Zoom.java

```diff
@@ -1656,13 +1660,31 @@ public void onMeetingUserJoin(List<Long> list) {
 
         // Schedule a task to run after a specified duration and check the number of participants to identify call missed/ignored use case
         if (inMeetingUserListSize == 1) { // patient is the only one who has joined
-            final Runnable runnable = new Runnable() {
-                @Override
-                public void run() {
-                    checkCallIgnoredByParticipant();
-                }
-            };
-            callIgnoredHandler.postDelayed(runnable, CALL_IGNORED_DIALOG_SHOW_AFTER_MILLIS); // Show after 90 seconds as this is the ringing time at clinician/caregivers end
+            if (declinedCallId != null) {  // Check whether the call was declined before the user could join
```

Comment:
There might be edge case when the call is just getting started from ZoomSDK and just before that the call was declined from CC2. In this case, we have maintained a state and when the patient joins, we check that this call was declined or not. If it was declined and we have not shown the message yet, we show it and end the call.